specs/Standards/NonFungibleToken/Payout.md

```diff
@@ -107,4 +121,4 @@ In the future, the NFT contract itself may be able to place an NFT transfer is a
 
 ## Errata
 
```

Comment:
- Version `2.1.0` adds the `memo` parameter, which while used by `nft_transfer` in implementations that follow `2.0.0`, would have encoded used `None`. It also changes `max_len_payout` to an option type.